funktionale-try/src/test/kotlin/org/funktionale/tries/TryTest.kt

```diff
@@ -59,6 +59,11 @@ class TryTest {
         failure.foreach { fail() }
     }
 
+    @Test fun onEach() {
```

Comment:
Yes, I can rewrite tests for `onEach` (and for `foreach` maybe too) a little bit.

Do you prefer amended commit or a new one? I am a fan of squashing irrelevant changes into one "feature" commit, but maybe you prefer otherwise or it's more convinine to work with separate commits on GitHub (I'm not a GitHub Pull Request veteran ;-) )